src/_data/env.js

```diff
@@ -4,5 +4,6 @@ require("dotenv").config();
 
 module.exports = {
     context: process.env.CONTEXT || "dev",
-    wordleRepo: "https://github.com/inclusive-design/inverted-wordles"
+    repo_owner: process.env.WORDLES_REPO_OWNER,
+    repo_name: process.env.WORDLES_REPO_NAME
```

Comment:
These two variables are exported for defining `wordle_globals` at the client side. Since it's found out fetching the answers file at the client side is inapplicable, these lines can be removed as well as from the `wordle_globals` object.

Reply:
Thanks, good catch and I've written up the old approach at https://issues.fluidproject.org/browse/FLUID-6626 and linked to it